migration/1638386837934-FixAnonymousDonationBoolean.ts

```diff
@@ -0,0 +1,24 @@
+import { MigrationInterface, QueryRunner } from 'typeorm';
+
+export class FixAnonymousDonationBoolean1638386837934
+  implements MigrationInterface
+{
+  public async up(queryRunner: QueryRunner): Promise<void> {
+    await queryRunner.query(
+      'UPDATE donation SET anonymous = true WHERE "userId" IS NULL',
+    );
+    await queryRunner.query(
+      'UPDATE donation SET anonymous = false WHERE "userId" IS NOT NULL',
+    );
+  }
+
+  // Revert in case something happens, seems only the new design will use this
+  public async down(queryRunner: QueryRunner): Promise<void> {
+    await queryRunner.query(
+      'UPDATE "donation" SET anonymous = false WHERE "userId" IS NULL',
```

Comment:
@CarlosQ96  I think you should set anonymous `true` in this case

Reply:
I was thinking what would be the down method in this case? I was like reverting it to how it was  before. The up method is like you said.

Reply:
Oh man sorry , my mistake
You are totally right